schema/common/defaults/configuration/opensearch.yml

```diff
@@ -16,7 +16,7 @@ specification:
     - readall
     - snapshotrestore
     # - logstash
-    # - kibanaserver
+    # - kibanaservers
```

Comment:
Changed by purpose? :) I see `kibanauser` in https://opensearch.org/docs/latest/security-plugin/configuration/yaml#internal_usersyml

Reply:
Probably added by accident.